packages/api/src/command/internal/update-hie-enabled-feature-flags.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  getFeatureFlags,
+  createAndDeployHieConfigurationContent,
+  CxFeatureFlagStatus,
+} from "@metriport/core/external/aws/app-config";
+import { Config } from "../../shared/config";
+
+export async function updateHieEnabledFFs({
+  cxId,
+  cwEnabled,
+  cqEnabled,
+}: {
+  cxId: string;
+  cwEnabled?: boolean;
+  cqEnabled?: boolean;
+}): Promise<CxFeatureFlagStatus> {
+  const region = Config.getAWSRegion();
+  const appId = Config.getAppConfigAppId();
+  const configId = Config.getAppConfigConfigId();
+  const envName = Config.getEnvType();
+  const envId = Config.getAppConfigEnvironmentId();
+  const deploymentStrategyId = Config.getAppConfigDeploymentStrategyId();
+  const featureFlags = await getFeatureFlags(region, appId, configId, envName);
+  if (cwEnabled === true) {
+    featureFlags.cxsWithCWFeatureFlag.values.push(cxId);
+  } else if (cwEnabled === false) {
+    featureFlags.cxsWithCWFeatureFlag.values = featureFlags.cxsWithCWFeatureFlag.values.filter(
+      id => id !== cxId
+    );
+  }
+  if (cqEnabled == true) {
+    featureFlags.cxsWithCQDirectFeatureFlag.values.push(cxId);
+  } else if (cqEnabled === false) {
+    featureFlags.cxsWithCQDirectFeatureFlag.values =
+      featureFlags.cxsWithCQDirectFeatureFlag.values.filter(id => id !== cxId);
+  }
```

Comment:
We have two types of FF, one that's a simple boolean and one that has an array of IDs. We could move these conditionals to a single function that we call w/ the FF, like:

```ts
  if (cqEnabled == true) {
    enableFeatureFlagForCustomer(featureFlags.cxsWithCQDirectFeatureFlag, cxId);
  } else {
    disableFeatureFlagForCustomer(featureFlags.cxsWithCQDirectFeatureFlag, cxId);
  }
```

Reply:
Done.